src/support/import-supervisor.js

```diff
@@ -110,6 +110,18 @@ function ImportSupervisor(services, config) {
     return dataAccess.createNewImportJob(newJob);
   }
 
+  /**
+   * Get all import jobs between the specified start and end dates.
+   * @param {string} startDate - The start date of the range.
+   * @param {string} endDate - The end date of the range.
+   * @param {string} importApiKey - The API
+   * @returns {Promise<Array<ImportJobDto>>}
+   */
+  async function getImportJobsByDateRange(startDate, endDate) {
+    const jobs = await dataAccess.getImportJobsByDateRange(startDate, endDate);
+    return jobs;
```

Comment:
Is this function necessary? 
```suggestion
    return dataAccess.getImportJobsByDateRange(startDate, endDate);
```